hail/src/main/scala/is/hail/expr/ir/InAgg.scala

```diff
@@ -0,0 +1,43 @@
+package is.hail.expr.ir
+
+object InAgg {
```

Comment:
This bottoms out at relational IR which seems wrong.  (Not every IR in the entire pipeline will get computed.)  I think things used in the optimizer should be whole pipeline.